### ISSUE TITLE
Is your fork stable / production ready?

### ISSUE THREAD

**Issue by sabiland** [kkieffer/ALCameraViewController#3]
I have had issues with latest ALCameraViewController and tried your fork. It seems everything works ok.
So, you will continue to develop ALCameraViewController? :)

**Comment by sabiland** [comment]
FYI, in simulator it still crashes:

![Simulator](https://i.imgur.com/e1AwoI8.png)

**Comment by kkieffer** [comment]
Yes, I have done a lot of testing with this fork. I will continue to support it.

**Comment by sabiland** [comment]
Great thx for info. I have already used your fork in one of my apps:)

I have quoted both Git repositories in credits (original and yours).
